### PR TITLE
Tagging Assets in OMD

### PR DESCRIPTION
**Pull request by danagerous**
This is documentation and scripts used to tag the first batch of assets. 

The primary purpose of the script to tag assets to an app based on tags already created in OpenMetadata.

### CODE REVIEW

File: scripts/openmetadata-tagging-project/src/db_connection_cx.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+    # Mask password for logging
+    def mask_password(password):
+        return '*' * len(password) if password else None
+
+    # Log environment variables (except password)
+    logging.info(f"ORACLE_DRIVER: {env_vars.get('ORACLE_DRIVER')}")
+    logging.info(f"ORACLE_DSN: {env_vars.get('ORACLE_DSN')}")
+    logging.info(f"ORACLE_USER: {env_vars.get('ORACLE_USER')}")
+    logging.info(f"ORACLE_PASSWORD: {mask_password(env_vars.get('ORACLE_PASSWORD'))}")
```
[the remark this answers is not in the document]
Comment (danagerous) [reply scripts/openmetadata-tagging-project/src/db_connection_cx.py]: Removed from database script.

File: scripts/openmetadata-tagging-project/README.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+1. Clone the repository
+   ```
+   git clone https://github.com/yourusername/openmetadata-tagging-project.git
+   cd openmetadata-tagging-project
+   ```
+
+2. Install dependencies:
+   ```
+   pip install -r requirements.txt`
```
Comment (Krishna-R-BC) [inline scripts/openmetadata-tagging-project/README.md:55]: We're missing the requirements.txt file

Comment (danagerous) [reply scripts/openmetadata-tagging-project/README.md:55]: Totally missed those in the initial commit. Included pipfiles too.

**Review by Krishna-R-BC — APPROVED** [review]
Thank you for adding the requirements.txt and the Pipfile!

**Pull request merged**